### PR TITLE
using dtl logger for dotnetcore build (#12358)

### PR DESCRIPTION
**Pull request by tejasd1990**
* using dtl logger for dotnetcore build

* removing failing L0 for now, will restore later

* removing logdetail

* removing msbuild args, cleanup

* cleanup

* incrementing task version

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/Tests/L0.ts [hunk 1]
```diff
@@ -205,18 +205,6 @@ describe('DotNetCoreExe Suite', function () {
         done();
     });
 
-    it('build passes when zero match found with empty string', (done: MochaDone) => {
```
Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreCLIV2/Tests/L0.ts]: why is this removed? and when will you add it back?

Comment (tejasd1990) [reply Tasks/DotNetCoreCLIV2/Tests/L0.ts]: https://github.com/microsoft/azure-pipelines-tasks/pull/12358#discussion_r386461029
The tests run with Mocha, so I will find out whether they can accept variable paths. I will add a user story to track.

Comment (bishal-pdMSFT) [reply Tasks/DotNetCoreCLIV2/Tests/L0.ts]: The way to make this work is to have a function (getLoggerPath()) which gets logger path and then mock that function to return a static path ("/dummy/logger/path") in test. this way the build command remain static and easy to validate.
